pyrk/driver.py

```diff
@@ -215,12 +235,26 @@ def main(args, curr_dir):
     logger.set_up_pyrklog(args.logfile)
     infile = load_infile(args.infile)
     out_db = database.Database(filepath=args.outfile)
+    if not hasattr(infile, 'n_ref'):
+        n_ref = 0
+        ref_lambda = 0
+        ref_rho = 0
+        Lambda_ref = 0
+    else:
+        n_ref = infile.n_ref
+        ref_lambda = infile.ref_lambda
+        ref_rho = infile.ref_rho
+        Lambda_ref = infile.Lambda_ref
```

Comment:
The above if else statement is not how we should handle default values from the input file in the long term. Let's think about the best way to do it. Probably having a more sophisticated input file syntax (such as json or similar) is the best way in the long term. However, in the short term, I think treating these 4 optional variables differently than the rest of the input variables is not as good as thinking about how to give the simulation better behavior for switching between models. Ultimately, the neutronics object is either single point or multipoint. If it's multipoint, then you want to pass in multipoint info, if single point, then we want to pass in single point info. Let's create a data structure that can hold either and make that object required in the input file. That way user can utilize their favorite model in the input file, only providing whatever information they need for that model.